### DISCUSSION TITLE
"Unsaved changes" indicator when tab bar is disabled

### DISCUSSION THREAD

**Discussion by Cre3per** [zed-industries/zed#15107] [Feature Requests]
When the tab bar (and left side bar) is disabled, is there any indicator that the active file is dirty?  
VSCode adds an indicator dot to the window title, which still is visible in that scenario. Could the same be done in zed? Alternatively, an indicator dot could be added to the breadcrumbs bar (The bar below the tab bar).
If there is no solution yet, I'd like to give the implementation a try

**Comment by dmarcoux** [comment]
I stumbled on this discussion while looking for an answer... and I did! It's been implemented in the PR https://github.com/zed-industries/zed/pull/30507. In short, the filename will be bold when it contains unsaved changes.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
